inginious/frontend/plugins/manual_scoring/static/js/rubric.js

```diff
@@ -33,18 +32,21 @@ class Rubric {
     addSelectFunction(matrixField) {
         const self = this;
         this.matrix[matrixField.iIndex][matrixField.jIndex].addEventListener("click", function () {
+            const isSelected = self.isSelected(matrixField);
```

Comment:
I do not like this `self`, it is kinda confusing for JS. You can rename the variable as _this or instead of doing `function() {` you can do `() => {` and it is not necessary to keep another variable for `this` as that preserves the context.